proto/v1/intoto_provenance.proto

```diff
@@ -71,12 +71,12 @@ message Metadata {
   bool reproducible = 5;
 }
 
-message Builder {
+message BuilderConfig {
```

Comment:
nit: maybe BuilderIdentity? since the submessage has the ID only, vs configuration of the builder. open to other ideas as well

Reply:
Named it that based on the comment:
// Identifies the configuration used for the build.
But happy to rename it. If it gets a new field, it may be more than Identity.

Reply:
sgtm, makes sense